app/common/patient/components/wizard/steps/patientAddressStep.js

```diff
@@ -5,7 +5,9 @@
     .module('common.patient')
     .component('patientAddressStep', {
       bindings: {
-        patient: '<'
+        patient: '<',
+        form: '<',
+        showMessages: '<'
       },
       controller: PatientAddressStepController,
       controllerAs: 'vm',
```

Comment:
Looks like `require: {patientWizard: '^^',}` bellow is no longer required.

Reply:
@edrisse All components in `app/common/patient/wizard/steps` seem to be have `require: {patientWizard: '^^',}`.

Reply:
Done @ynurmahomed